app/src/main/java/com/keylesspalace/tusky/interfaces/StatusActionListener.java

```diff
@@ -22,9 +22,12 @@
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 
+import com.keylesspalace.tusky.entity.Status;
+
 public interface StatusActionListener extends LinkListener {
     void onReply(int position);
     void onReblog(final boolean reblog, final int position);
+    void onReblog(final boolean reblog, final int position, final Status.Visibility visibility);
```

Comment:
Please convert this interface to Kotlin, then there can be only one `onReblog`, with `visibilty` optional.

Reply:
You need to provide a default parameter:
```kotlin
fun onReblog(reblog: Boolean, position: Int, visibility: Status.Visibility = Status.Visibility.PUBLIC)
```

That does not work for Java callers, but there aren't many and you can just pass PUBLIC again:
```java
listener.onReblog(false, position, Status.Visibility.PUBLIC);
```

Reply:
@connyduck I broke the problem down in one Kotlin file and I don't reproduce my issue, I'm not sure why it insists on wanting both methods. I only have Kotlin code implementing the interface.

I will push a temporary commit to illustrate the error I'm getting.

> e: file:///home/exagone/project/mobile/tusky/app/src/main/java/com/keylesspalace/tusky/components/conversation/ConversationsFragment.kt:68:1 Class 'ConversationsFragment' is not abstract and does not implement abstract base class member 'onReblog'.

Reply:
That is because `SFragment` also has a `onReblog`. The solution is to change the signature of the function in `SFragment` as well.

Reply:
@connyduck I updated it, let me know if there are other changes to make.